src/celluloid-application.c

```diff
@@ -307,6 +308,48 @@ options_handler(GApplication *gapp, GVariantDict *options, gpointer data)
 	return version?0:-1;
 }
 
+static gboolean
+local_cmdline_handler (	GApplication *gapp,
+			gchar ***arguments,
+			gint *exit_status)
+{
+	CelluloidApplication *app = CELLULOID_APPLICATION(gapp);
+	gchar **argv = *arguments;
+	g_clear_pointer(&app->mpv_options, g_free);
+
+	GArray *options = g_array_new(TRUE, FALSE, sizeof(gchar *));
+	gint i = 1;
+	while(argv[i])
+	{
+		if(g_str_has_prefix(argv[i], "--mpvopt-"))
+		{
+			gchar *option = g_strconcat("--", &argv[i][sizeof("--mpvopt-")-1], NULL);
+			g_array_append_val(options, option);
+
+			for(gint j = i; argv[j]; j++)
+			{
+				argv[j] = argv[j + 1];
+			}
+		}
+		else
+		{
+			i++;
+		}
+	}
+	if(options->len)
+	{
+		gchar **data = (gchar **)options->data;
+		app->mpv_options = g_strjoinv(" ", data);
+		for(gint i = 0; data[i]; i++)
+		{
+			g_free(data[i]);
+		}
+	}
+	g_array_free(options, TRUE);
+
+	return G_APPLICATION_CLASS(celluloid_application_parent_class)->local_command_line(gapp, arguments, exit_status);
```

Comment:
Long lines should be wrapped.
```suggestion
	return G_APPLICATION_CLASS(celluloid_application_parent_class)
	->local_command_line(gapp, arguments, exit_status);
```